chains/orchestrator-relays/runtime/dancelight/src/tests/ethereum_token_transfers.rs

```diff
@@ -1672,6 +1678,479 @@ fn test_pricing_parameters() {
+
+            // assert_ok!(EthereumTokenTransfers::set_token_transfer_channel(
+            //     root_origin(),
+            //     channel_id,
+            //     agent_id,
+            //     para_id
+            // ));
+
+            let amount_to_transfer = 10_000u128;
+
+            // Define a mock ERC20 token address
+            let token_address = H160(hex!("deadbeefdeadbeefdeadbeefdeadbeefdeadbeef"));
+
+            let erc20_asset_location = Location {
+                parents: 1,
+                interior: X2([
+                    GlobalConsensus(EthereumNetwork::get()),
+                    AccountKey20 {
+                        network: Some(EthereumNetwork::get()),
+                        key: token_address.into(),
+                    },
+                ]
+                .into()),
+            };
+
+            let asset_id = 42u16;
+
+            assert_ok!(ForeignAssetsCreator::create_foreign_asset(
+                root_origin(),
+                erc20_asset_location.clone(), // Use the ERC20 location
+                asset_id,
+                AccountId::from(ALICE),
+                true,
+                1
+            ));
+
+            // Give tokens to user as if tokens were bridged before + extra to check the correct
+            // amount is sent
+            ForeignAssets::mint_into(asset_id, &AccountId::from(BOB), amount_to_transfer + 10)
+                .expect("to mint amount");
+
+            // User tries to send tokens
+            let beneficiary_address = H160(hex!("0123456789abcdef0123456789abcdef01234567"));
+
+            // Beneficiary location must be represented using destination's point of view.
+            let beneficiary_location = Location {
+                parents: 0,
+                interior: X1([AccountKey20 {
+                    network: Some(EthereumNetwork::get()),
+                    key: beneficiary_address.into(),
+                }]
+                .into()),
+            };
+
+            let eth_asset = AssetId(erc20_asset_location.clone())
+                .into_asset(Fungibility::Fungible(amount_to_transfer));
+
+            assert_noop!(
+                XcmPallet::transfer_assets(
+                    RuntimeOrigin::signed(AccountId::from(BOB)),
+                    Box::new(EthereumLocation::get().into()),
+                    Box::new(beneficiary_location.into()),
+                    Box::new(vec![eth_asset].into()),
+                    0u32,
+                    Unlimited,
+                ),
+                pallet_xcm::Error::<Runtime>::SendFailure
+            );
+
+            assert_eq!(
+                filter_events!(RuntimeEvent::EthereumOutboundQueue(
+                    snowbridge_pallet_outbound_queue::Event::MessageQueued { .. },
+                ))
+                .count(),
+                0,
+                "MessageQueued event should NOT be emitted!"
+            );
+        })
+}
+
+#[test]
+fn cant_send_tokens_of_unknown_remote_network() {
```

Comment:
Can we add a test in which we send a message not related to to ethereum token transfers (through pallet-xcm->send), I dont know something like a clearOrigin and see what happens?